mainBoard.py

```diff
@@ -0,0 +1,295 @@
+import shipClass
+import score_class
+
+import os
+import sys
+import pygame
+from pygame.locals import *
+import random
+import math
+if not pygame.font:
+    print('Warning, fonts disabled')
+if not pygame.mixer:
+    print('Warning, sound disabled')
+
+
+class BoardInit:
+    def rotShip(self, myangle, shipNum):
+        """rotate the main ship"""
+        self.ships[shipNum].rotatecw(angle=myangle)
+        """changes the ship object in the list of ships"""
+        self.shipImages[shipNum] = rot_center(self.shipImage, self.ships[shipNum].get_angle_from_vert())
+        """this will call the function to rotate the ship image about its center"""
+        self.shiprects[shipNum] = self.shipImages[shipNum].get_rect()
+        """this will grab the bounding box of the new ship"""
```

Comment:
I think this is an example of overcommented code - you don't need to document every single line unless they are particularly confusing.